src/evict/evict_lru.c

```diff
@@ -1555,15 +1555,6 @@ __evict_walk(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue)
             continue;
         }
 
-        /*
-         * If we are filling the queue, skip files that haven't been useful in the past.
-         */
-        if (btree->evict_walk_period != 0 && btree->evict_walk_skips++ < btree->evict_walk_period) {
-            WT_STAT_CONN_INCR(session, cache_eviction_server_skip_trees_not_useful_before);
```

Comment:
As the stat is not used any more, remove the stat also.

Reply:
Nice catch - updated in [c3679c2](https://github.com/wiredtiger/wiredtiger/pull/10283/commits/c3679c22cd1df8d6ae905dccea97c50c53834726)